vue/src/components/SnapshotMap.vue

```diff
@@ -9,137 +9,56 @@
-          <div id="PostitNaming" v-if="addMarkerMode
-                  && markerSelection.includes('tooltip')">
-            <div class="row">
-              <div class="column" style="width: 150px;">
-                <h6> Enter node & <br>click on the map.</h6>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-green'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="green" > mdi-tooltip-text </v-icon>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-blue'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="blue" > mdi-tooltip-text </v-icon>
-              </div>
-              <div v-if="markerSelection == 'tooltip-text-yellow'"
-              class="column" style="width: 50px;"
-              v-on:click="addMarkerMode = !addMarkerMode;
-              markerSelection = ''">
-                <v-icon large color="yellow" > mdi-tooltip-text </v-icon>
-              </div>
-              <div class="column" style="width: 150px; font-size: 1.4em">
-                <input id="InputMarkerName"  v-model="newPostItNode"
-                    placeholder="Enter your Post-it note here"
-                >
-              </div>
-            </div>
-          </div>
+      <v-main class="pa-0" ref="mapbox">
+      <div
+        id="MarkerButtons"
+        style="position: absolute; top: 6em; right: 1.8em; z-index: 500"
+      >
+        <button
+          style="background-color: #543076; border-radius: 50px"
+          v-on:click="markerTools=!markerTools; showAnnotations(); map.closePopup();"
+        >
+          <v-icon large color="white"> mdi-brush </v-icon>
+        </button>
+        <v-card v-if="markerTools" style="position:absolute: ;width: 36px">
+          <button v-on:click="toggelMarkerSelection('marker')">
+            <v-icon large color="black"> mdi-map-marker </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection('polygon')">
+            <v-icon large color="black"> mdi-vector-polygon </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection('note')">
+            <v-icon large color="black"> mdi-note-outline </v-icon>
+          </button>
+          <button v-on:click="toggelMarkerSelection(''); hideAnnotations();
+          markerTools=!markerTools;">
+            <v-icon large color="black"> mdi-eye-off </v-icon>
+          </button>
+        </v-card>
+        <div v-if="markerTools" >
+          <v-card v-if="markerSelection == 'marker'"
+          style="position:absolute; top:3.8em; right:3.7em; min-width:16em"
+          color="rgba(255, 255, 255, 0.7)"
+          >Klicken Sie auf die Karte, um eine Markierung zu platzieren </v-card>
```

Comment:
We should extract the into [vue-i18n](https://vue-i18n.intlify.dev). This would need a block at the top of the file looking like this:

```js
<!-- eslint-disable -->
<i18n>
{
  "de": {
    "tooltip.marker": "Klicken Sie auf die Karte, um eine Markierung zu platzieren",
    "tooltip.polygon": "Klicken Sie auf die Karte, um ein Polygon zu beginnen"
  },
  "fr": {
    "tooltip.marker": "FRENCH VERSION",
    "tooltip.polygon": "FRENCH VERSION"
  }
}
</i18n>
<!-- eslint-enable -->
```

and replacing the strings with

```js
<v-tooltip>{{ $t("tooltip.marker") }}</v-tooltip>
```

Reply:
https://github.com/cividi/spatial-data-package-platform/commit/54bda44f9426c5ad9d630b0150147369e47084a6 do the magic with internationalization